deployments/kube-deployment.yml

```diff
@@ -120,6 +137,9 @@ spec:
         - name: kubi-tls
           secret:
             secretName: kubi
+        - name: kubi-encryption
```

Comment:
Is this mandatory?

Reply:
yes otherwise it will not be able to find the secret